Jenkinsfile.nightlyadhoc

```diff
@@ -175,33 +176,50 @@ stage('Build Applications'){
     parallel stepsForParallelBuilds
 }
 
+def transitionIssueList
+def passedBuilds
+stage('JIRA'){
+    withEnv(['JIRA_SITE=cliqztix']){
+        def returnList = jira.transitionIssues()
+        passedBuilds = returnList[0]
+        transitionIssueList = returnList[1]
+    }
+}
+
 stage('Send email') {
     def projectName  = '$PROJECT_NAME'
+    def changesSinceLastSuccess = jira.getChangeString(passedBuilds)
     def bodytxt = """
         ${projectName} - Build # ${env.BUILD_NUMBER}
         Check console output at ${env.JOB_URL}${env.BUILD_NUMBER}/ to view the results.
         """
     params.each { prop, val ->
-        if(val){
-            def splitText = prop.toString().toLowerCase().split("_")
-            def brand = ""
-            def suffix = ""
-            if (splitText.size()==3){
-                if (splitText[2] == "x86"){
-                    suffix = "_x86"
-                }
-                brand = "${splitText[0]}-${splitText[1]}"
-            } else {
-                if (splitText[1] == "x86"){
-                    suffix = "_x86"
+        if(prop != "BRANCH"){
```

Comment:
this will fail easily when we add new param. How about we test for some pattern? or add a common pre/suffix for params we care?